clang/lib/Sema/SemaTemplate.cpp

```diff
@@ -2744,31 +2744,155 @@ bool hasDeclaredDeductionGuides(DeclarationName Name, DeclContext *DC) {
   return false;
 }
 
+unsigned getTemplateDepth(NamedDecl *TemplateParam) {
```

Comment:
I think this is potentially misleading, as we already have multiple unrelated `getTemplateDepth`, see `Sema.h` / `SemaTemplate.cpp`, `DeclBase.h`, `Expr.h`.

Reply:
Renamed to `getTemplateParameterDepth` per your suggestion.